layers/+tools/shell/packages.el

```diff
@@ -291,3 +290,23 @@
                             eshell-mode-hook
                             shell-mode-hook
                             term-mode-hook)))
+
+(defun shell/init-vterm ()
+  (use-package vterm
+    :defer t
+    :commands (vterm vterm-other-window)
+    :init
+    (progn
+      (make-shell-pop-command "vterm" spacemacs/open-vterm)
+      (spacemacs/set-leader-keys "asv" 'spacemacs/shell-pop-vterm)
+      (spacemacs/register-repl 'vterm 'vterm))
+    :config
+    (progn
```

Comment:
is there a reason for these bindings?

Reply:
It is a way of being able to use <RET> or <ESC> in normal mode, without having to go to insert mode. I found it convenient at the time as I was using a lot of interactive command line utilities, but now I haven't used it for a long time so I guess it is not _that_ important.

Reply:
Removed the extra bindings ✨